tests/urls.py

```diff
@@ -2,5 +2,6 @@
 from django.conf.urls import include, url
 
 urlpatterns = [
-    url(r'^accounts/', include('nopassword.urls', namespace='nopassword')),
```

Comment:
Backward incompatible: Namespace not used anymore (deprecates `NOPASSWORD_NAMESPACE`)